cmake/ElektraConfig.cmake.in

```diff
@@ -46,3 +43,81 @@ set (Elektra_BUILD_TYPE @Elektra_BUILD_TYPE@)
 set (Elektra_SHARED @BUILD_SHARED@)
 set (Elektra_FULL @BUILD_FULL@)
 set (Elektra_STATIC @BUILD_STATIC@)
+
+# ~~~
+# Invoke the `kdb gen` during the build.
+# 
+# elektra_kdb_gen (template parent_key output_name
+#                  INPUT <plugin> <file>
+#                  OUTPUT_DIR <output_dir>
+#                  KDB <kdb>
+#                  PARAMS <params>...
```

Comment:
Isn't PARAMS optional?

Reply:
Yes, and `OPTIONS`, `KDB` and `OUTPUT_DIR` too. I'll make that more clear.